app/services/trending_groups.rb

```diff
@@ -0,0 +1,26 @@
+class TrendingGroups
```

Comment:
I was gonna do that but there was something that got in my way.  I'll look into it again.